plugins/catalog-backend/migrations/20221201085245_add_last_updated_at_in_final_entities.js

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+exports.up = async function up(knex) {
+  await knex.schema.table('final_entities', table => {
+    table
+      .bigint('last_updated_at')
+      .nullable()
```

Comment:
Yeah, there are 2 options here,
Option 1. last_updated_at with knex.fn.now() will auto-fill it for all entities, but the value will be incorrect because these entities are not last_updated_at this time.
Option 2. last_updated_at with null will initiate a null last_updated_at, but it can indicate last_updated_at for those entities are unknown yet.
To be honest, i feel Option 2 is better. 
Please let me know if my understanding is correct.

Reply:
Yeah you understood it right. And I realized that no matter what, the column itself HAS to be nullable since we do create the row before the stitching is complete just to insert the stitch ticket. And the stitching may fail, so the column may stay as null for that reason as well.

It's still unfortunate that things that have been properly stitched may have null there "forever" if their inputs never change so they don't get restitched. But OK, let's pick that path.